README.md

```diff
@@ -1,3 +1,4 @@
+<!-- vim: set spell spelllang=en: -->
```

Comment:
should be possible to move this to the bottom for less clutter when reading this as a text file?